portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesConsentPanel.testcase

```diff
@@ -2,7 +2,8 @@
 definition {
 
 	property ci.retries.disabled = "true";
-	property custom.properties = "feature.flag.LPS-142518=true";
+	property custom.properties = "feature.flag.LPS-142518=true${line.separator}jsonws.web.service.paths.excludes=";
```

Comment:
@melodywu I think you wanted to add this custom.poperty to the other test case file?

Reply:
done